interp/interp.go

```diff
@@ -148,13 +147,14 @@ type interp struct {
 	ctxOps   int
 
 	// Misc pieces of state
-	random           *rand.Rand
-	randSeed         float64
-	exitStatus       int
-	regexCache       map[string]*regexp.Regexp
-	formatCache      map[string]cachedFormat
-	csvJoinFieldsBuf bytes.Buffer
-	chars            bool
+	random            *rand.Rand
+	randSeed          float64
+	exitStatus        int
+	regexCache        map[string]*regexp.Regexp
+	formatCache       map[string]cachedFormat
+	csvJoinFieldsBuf  bytes.Buffer
+	chars             bool
+	newlineOutputCRLF bool
```

Comment:
Nice. Let's also use this in `writeCSV`, where we currently do `writer.UseCRLF = runtime.GOOS == "windows"` -- that should really use the same setting.